src/credentials/tests/TestPersistentStorageDelegate.h

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
```

Comment:
This file does not belong under src/credentials/tests.

I would suggest putting this under lib/support and adding a new build target in lib/support/BUILD.gn for the `TestPersistentStorageDelegate` that could be included by more unit tests.

Reply:
File moved.